.github/workflows/go.yaml

```diff
@@ -40,7 +40,6 @@ jobs:
                --enable stylecheck \
                --enable interfacer \
                --enable unconvert \
-               --enable dupl \
```

Comment:
Why remove this?

Reply:
startObservations and startJetStreamAdvisories is almost identical and it moans, and yes its almost identical and its a legit gripe, but generic helper would be more complicated and I'd say if we get a 3rd similar its worth extracting into a helper, for 2 its just not worth it

Reply:
+1